yt_dlp/extractor/amcnetworks.py

```diff
@@ -85,7 +85,7 @@ def _real_extract(self, url):
                     media_url = 'https://link.theplatform.com/s/' + tp_path
                     has_releasePid = True
                     video_player_count += 1
-        except:
+        except KeyError:
```

Comment:
don't try...except. Use `try_get` or `.get` when accessing keys that are not necessarily present

Reply:
That'd have more complicated syntax and have more function calls than the way I wrote it.

Reply:
see my suggestion